src/Cornelis/Types/Agda.hs

```diff
@@ -103,26 +84,10 @@ instance Show a =>
               ((.)
                  showSpace (showsPrec 11 $ toList b2_a1hOm))))
 
-data Interval' a = Interval { iStart, iEnd :: !(Pos' a) }
-  deriving (Data, Eq, Ord, Functor, Foldable, Traversable, Generic)
-
-instance Show a => Show (Interval' a) where
```

Comment:
I'd like to keep this show instance for the new `Interval`; a short-term goal is to send intervals back to Agda for improving the reload speed, and this is the format it parses as.